core/trino-main/src/test/java/io/trino/operator/join/unspilled/TestHashJoinOperator.java

```diff
@@ -17,7 +17,6 @@
 import com.google.common.collect.ImmutableSet;
```

Comment:
do we have any multi channel tests left?

Reply:
Yes, there are 4 or 5